go/vt/vttablet/common/flags.go

```diff
@@ -0,0 +1,104 @@
+/*
+Copyright 2024 The Vitess Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package vttablet
+
+import (
+	"time"
+
+	"github.com/spf13/pflag"
+
+	"vitess.io/vitess/go/vt/servenv"
+)
+
+const (
+	// VReplicationExperimentalFlags is a bitmask of experimental features in vreplication.
+	VReplicationExperimentalFlagOptimizeInserts           = int64(1)
+	VReplicationExperimentalFlagAllowNoBlobBinlogRowImage = int64(2)
+	VReplicationExperimentalFlagVPlayerBatching           = int64(4)
+)
+
+var (
+	// Default flags.
+	vreplicationExperimentalFlags   = VReplicationExperimentalFlagOptimizeInserts | VReplicationExperimentalFlagAllowNoBlobBinlogRowImage
```

Comment:
Can you add a comment for how this is computed? Why is `VReplicationExperimentalFlagVPlayerBatching` not included? I'm inferring that the default for it is false, but it will be good to write that down.